hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducerListener.java

```diff
@@ -115,6 +131,16 @@
     public void onPublishComplete(ProducerStatus status, long elapsed, TimeUnit unit);
 
     /**
+     * Called once a blob has been published successfully or failed to published. Use {@link PublishStatus#getBlob()} to get more details on blob type and size.
+     * This method is called for every {@link com.netflix.hollow.api.producer.HollowProducer.Blob.Type} that was published.
+     *
+     * @param publishStatus Status of publishing. {@link PublishStatus#getStatus()} returns {@code SUCCESS} or {@code FAIL}.
+     * @param elapsed       time taken to publish the blob
+     * @param unit          unit of elapsed.
+     */
+    public void onBlobArtifactPublishComplete(PublishStatus publishStatus, long elapsed, TimeUnit unit);
```

Comment:
Let's reserve `*Complete` for notifications that have a companion `*Start`.

Can maybe omit `Blob` from the name.